universite-de-sherbrooke-histoire.csl

```diff
@@ -138,25 +144,18 @@
           <text variable="container-title" font-style="italic" suffix=", "/>
         </group>
       </else-if>
-      <else-if type="report song" match="any">
+      <else-if type="interview graphic">
```

Comment:
```suggestion
      <else-if type="interview graphic" match="any">
```
Always need to set match=any when testing for Item types (default is "all" which is impossible)